.gitignore

```diff
@@ -12,6 +12,9 @@ out/
 
 # VS Code
 .vscode
```

Comment:
I found it useful to exchange some vscode settings in the project. Like markdown line length or so. unfortunately some settings only available as vscode setting, not editorconfig